package.json

```diff
@@ -192,8 +193,8 @@
     "winston-transport": "^4.5.0"
   },
   "optionalDependencies": {
-    "@newrelic/native-metrics": "^9.0.0",
-    "@contrast/fn-inspect": "^3.3.0"
+    "@contrast/fn-inspect": "^3.3.0",
+    "@newrelic/native-metrics": "^9.0.0"
```

Comment:
Funny irrelevant change. I guess npm did this automatically?

Reply:
It did, it tries to sort things alphabetically, my bet is that the contrast module was manually added (as opposed to with `npm`)